jenkins_job_wrecker/modules/triggers.py

```diff
@@ -1,4 +1,6 @@
 # encoding=utf8
+from builtins import NotImplementedError
```

Comment:
builtins is not available in Python 2.7 (tox confirms this)

in any case you don't need it - it works without importing NotImplementedError explicitly